src/languageservice/parser/ast-converter.ts

```diff
@@ -138,6 +142,18 @@ function convertScalar(node: Scalar, parent: ASTNode): ASTNode {
       result.isInteger = Number.isInteger(result.value);
       return result;
     }
+    default: {
+      if (node.tag === 'tag:yaml.org,2002:binary') {
```

Comment:
Why do we need this `if` in here? It looks like we are constructing exactly the same result to return?